### PR TITLE
tuna/example/load_job - type definitions added

### PR DESCRIPTION
**Pull request by technicalgrp89**
mypy check compatibility
ypaf,pylint compatible code
mypy command is added in to utils.groovy

### CODE REVIEW

**Review by reidkwja — APPROVED** [review]

File: tuna/example/load_job.py [hunk 1]
```diff
@@ -53,19 +56,19 @@ def parse_args():
...
-  args = parser.parse_args()
+  args: argparse.Namespace = parser.parse_args()
   if not args.session_id:
     parser.error('session_id must be specified')
 
   return args
 
 
-def add_jobs(args, dbt):
+def add_jobs(args: argparse.Namespace, dbt: ExampleDBTables) -> int:
```
Comment (msaudulhassan) [inline tuna/example/load_job.py]: It's seems to be outside the scope of this PR, but passing around `args` doesn't seem ideal. It's unclear what args.* the function needs without going through the code. Had the signature been `def add_jobs(label: str, session_id: int, dbt: XYZ)`, it'd be clear outright what the function needs. Or, perhaps, one can create `TunaNamespace` as a subclass of argparse.Namespace, and pass that around? @cderb

Comment (msaudulhassan) [inline tuna/example/load_job.py]: `dbt` seems to be `typing.Type[ExampleDBTables]`

**Review by msaudulhassan — CHANGES_REQUESTED** [review]

File: tuna/example/load_job.py (hunk 1 above)
Comment (technicalgrp89) [reply tuna/example/load_job.py]: @alexandraBara

Comment (cderb) [reply tuna/example/load_job.py]: @msaudulhassan fair enough, but should mark that out of scope. Perhaps @technicalgrp89 can add a comment to the function that this update should be made for clarity.

Comment (technicalgrp89) [reply tuna/example/load_job.py]: I will add the comment

**Review by msaudulhassan — APPROVED** [review]

**Review by cderb — APPROVED** [review]

**Pull request merged**